recipes/mne-python_1.0/construct.yaml

```
@@ -25,7 +25,8 @@ default_prefix_all_users: "%ALLUSERSPROFILE%\\mne-python\\1.0.3_0"  # [win]
 
 uninstall_name: MNE-Python ${VERSION} (Python ${PYVERSION})
 
-installer_filename: MNE-Python-1.0.3_0-macOS_Intel.pkg  # [osx]
+installer_filename: MNE-Python-1.0.3_0-macOS_Intel.pkg  # [osx and not arm64]
+installer_filename: MNE-Python-1.0.3_0-macOS_ARM.pkg    # [osx and arm64]
```

Errant change from last commit
```suggestion
installer_filename: MNE-Python-1.0.3_0-macOS_M1.pkg    # [osx and arm64]
```